core/core.py

```diff
@@ -1,73 +1,277 @@
+                    "metrics": {
+                        "codelines": 20000,
+                        "admins": 10,
+                        "recovery_time": 5
+                    }
+                },
+                {
+                    "uid": "b141f94973a43cf8ee972e9dffc1b004",
+                    "weight": 1.5,
+                    "name": "Frontend API",
+                    "category": "API",
+                    "description": "API für das Frontend",
+                    "creation_timestamp": "20200219...",
+                    "last_timestamp": "20200219...",
+                    "metrics": {
+                        "codelines": 20000,
+                        "admins": 10,
+                        "recovery_time": 5
+                    }
+                },
+                {
+                    "uid": "b141f94973a43cf8ee972e9dffc1b004",
+                    "weight": 2,
+                    "name": "Hadoop Cluster",
+                    "category": "Datenbank",
+                    "description": "Big Data Plattform",
+                    "creation_timestamp": "20200219...",
+                    "last_timestamp": "20200219...",
+                    "metrics": {
+                        "codelines": 20000,
+                        "admins": 10,
+                        "recovery_time": 5
+                    }
+                }
+            ]
+        },
+        "should-metrics": {
+            "codelines": 25000,
+            "admins": 12,
+            "recovery_time": 3
+        },
+        "score": 80,  # percent as integer
+        "actual_metrics": {
+            "codelines": {
+                "value": 30,
+                "fulfillment": True
+            },  # true means that the metric is fulfilled --> no problem.
+            "admins": {
+                "value": 30,
+                "fulfillment": True
+            },
+            "recovery_time": {
+                "value": 20,
+                "fulfillment": False
+            },  # false means that the metric is not fulfilled --> problem.
+        },
+    }
+    
+    return processing.dict_to_json(data)
+
+
+def create_edit_process(input_object: Union[Dict[str, Any], str]) -> str:
+    """
+    Receives a JSON object in the form defined under JSON_objects_defitnions.py for either editing a process or creating a new process
+    The answer is also a JSON object containing either the success state if False, otherwise calls get_process
+    
+    :param input_object: JSON object containing all process attributes (special information to differentiate edit or create is contained in the UID, which is either -1 or the original UID
+    :type input_object: str
+    :return: A JSON object containing either the success state if False, otherwise calls get_process
+    """
+
+    data_dict = processing.json_to_dict(input_object)
+
+    if data_dict["uid"] == "-1":
+        result_dict = process_handler.add_process(data_dict)
+        if result_dict["success"]== True:
```

Comment:
Expression can be simplified

```suggestion
        if result_dict["success"]:
```